shenyu-client/shenyu-client-http/shenyu-client-springmvc/src/main/java/org/apache/shenyu/client/springmvc/init/ContextRegisterListener.java

```diff
@@ -90,11 +95,21 @@ public void onApplicationEvent(@NonNull final ContextRefreshedEvent contextRefre
         if (isFull) {
             publisher.publishEvent(buildMetaDataDTO());
         }
+        // if mvc project, wait two seconds for metadata to register first,
```

Comment:
whats mean？

Reply:
for mvc project, uri is not register success on first start, need to restart project to register uri success